app/src/main/java/com/cpr3663/cpr_scouting_app/PreMatch.java

```diff
@@ -113,31 +137,14 @@ protected void onCreate(Bundle savedInstanceState) {
             // If we got match data, get the list of teams from it
             if (Match != null) {
                 ArrayList<String> teamsInMatch = Match.getListOfTeams();
-                boolean found = false;
-                int position = 0;
-
-                // See if there's a current team to scout (can happen if we hit the BACK button from Match
-                if (Globals.CurrentTeamToScout > 0) {
-                    // See if the current team is already in the list
-                    for (String ct : teamsInMatch) {
-                        if (ct.equals(String.valueOf(Globals.CurrentTeamToScout))) {
-                            found = true;
-                            break;
-                        }
-                        else position++;
-                    }
-
-                    // If we didn't find the team we're scouting in the list of teams, add it (it was an override)
-                    if (!found) teamsInMatch.add(String.valueOf(Globals.CurrentTeamToScout));
-                }
 
                 // Create and apply the adapter to the spinner.
                 ArrayAdapter<String> adapter = new ArrayAdapter<String>(this, R.layout.cpr_spinner, teamsInMatch);
                 adapter.setDropDownViewResource(R.layout.cpr_spinner_item);
                 spinner_Team.setAdapter(adapter);
 
                 // Set the spinner to the right selection
-                if (found) spinner_Team.setSelection(position);
+                if (Globals.CurrentTeamToScout > 0) spinner_Team.setSelection(Globals.CurrentTeamToScout);
```

Comment:
setSelection() takes the POSITION (ie: 0-6) as input, not the text (teamnum).  Why did you change this?  It worked before.